tests/install_upgrade_operators/hco_enablement_golden_image_updates/test_update_hco_cr.py

```
@@ -88,18 +87,13 @@ def test_add_custom_data_import_cron_template(
-        reason=f"{QUARANTINED}: Teardown AssertionError in verify_boot_sources_reimported after re-enabling"
-        " enableCommonBootImageImport spec, CNV-88015",
-        run=False,
-    )
     @pytest.mark.dependency(name="test_add_custom_data_import_cron_template_disable_spec")
     @pytest.mark.polarion("CNV-7914")
     def test_add_custom_data_import_cron_template_disable_spec(
         self,
         admin_client,
         hco_namespace,
-        disabled_common_boot_image_import_hco_spec_scope_function,
+        disabled_boot_image_import_excluding_custom_datasource,
```

Ruff (0.15.15)

[warning] Unused method argument: `disabled_boot_image_import_excluding_custom_datasource`

(ARG002)